dll/shellext/deskadp/lang/pt-PT.rc

```diff
@@ -1,3 +1,10 @@
+/*
++ * PROJECT:     ReactOS Adapter Shell Extension
++ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
++ * PURPOSE:     Portuguese (Portugal) resource translation
++ * COPYRIGHT:   Copyright 2019 2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
++ */
```

Comment:
What's the benefit of all those useless pluses?

Reply:
And the missing minus between 2020 and 2024?

Reply:
My fault.